tests/dummy/app/helpers/reopen-instrumentation.js

```diff
@@ -0,0 +1,51 @@
+/* global heimdall */
```

Comment:
despite setting this globally many places ESLint was yelling at me if I didn't set it at the top of files :/ This should be worked out.